games-strategy/globulation2/globulation2-0.9.4.4.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="Innovative new Real Time Strategy game"
+DESCRIPTION="Globulation 2 is an innovative Real-Time Strategy (RTS) game which reduces \
+micromanagement by automatically assigning tasks to units."
+HOMEPAGE="https://globulation2.org/"
+COPYRIGHT="Globulation2 team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://mirror.netcologne.de/savannah/glob2/0.9.4/glob2-$portVersion.tar.gz"
+SOURCE_FILENAME="glob2-$portVersion.tar.gz"
+SOURCE_DIR="glob2-$portVersion"
+CHECKSUM_SHA256="0f4d898ec6b05ce27b4a12ef242cc26571304b90d2509932a4743c71311314b8"
+PATCHES="globulation2-0.9.4.4.patchset"
+ADDITIONAL_FILES="globulation2.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2 !x86"
```

Comment:
Changes:
```diff
- ARCHITECTURES="all !x86_gcc2"
+ SECONDARY_ARCHITECTURES="x86"
```

Reply:
I actually did do it like that previosly, it still looked for the libraries in the x86_gcc2 parts

Reply:
Although it still doesn't work the change would eliminate a build for 32bit (x86_gcc2 primary and x86 secondary architecture)